packages/hub/config/structure.sql

```diff
@@ -754,6 +778,35 @@ CREATE TABLE public.merchant_infos (
 
 ALTER TABLE public.merchant_infos OWNER TO postgres;
 
+--
+-- Name: notification_preferences; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.notification_preferences (
+    owner_address text NOT NULL,
+    notification_type_id uuid NOT NULL,
+    push_client_id text NOT NULL,
+    status public.notification_preferences_status_enum DEFAULT 'enabled'::public.notification_preferences_status_enum NOT NULL,
+    created_at timestamp without time zone DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
```

Comment:
Consider adding a unique index on the combination of owner_address, noification_type_id, and push_client_id

Reply:
Yeah, it's already added: https://github.com/cardstack/cardstack/pull/2442/commits/8448d98ff942fdd77c156a297f7a8db3083e2cfa#diff-c33f7dc267c12ec716a2fc35c636dd8bcde8de8efecf7a54d0c171075b9e7fbdR16